### DISCUSSION TITLE
Update @typescript-eslint packages to version 8 while keeping eslint at version 8

### DISCUSSION THREAD

**Discussion by dustydang-tyler** [typescript-eslint/typescript-eslint#9838] [Technical Discussions]
### Body

I have a question regarding the usage of @typescript-eslint/eslint-plugin and @typescript-eslint/parser version 8 with eslint version 8.57.0.

I received a lot of errors for "Definition for rule xx is not found". Do I have to migrate my eslint to version 9 while using @typescript-eslint packages at version 8?

Please can someone clarify for me? Thanks!

### Additional Info

![definition not found issue](https://github.com/user-attachments/assets/1d3df205-3ce2-4b4f-98fd-df20d0636f5a)

### Before you submit your discussion, please confirm the following. If any of these required steps are not taken, we may not be able to review your RFC. Help us to help you!

- [X] I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.
- [X] I have [searched for related discussions](https://github.com/typescript-eslint/typescript-eslint/discussions) and [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues) and found none that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

**Comment by kirkwaiblinger** [comment]
Hi @dustydang-tyler!

This is expected - Those rules have been removed from the typescript-eslint plugin! 🙂 See also the [v8 migration guide](https://typescript-eslint.io/blog/announcing-typescript-eslint-v8#rule-breaking-changes), specifically the entry on "formatting rules". You can continue to use them if you want over at https://eslint.style/. 

Note for anyone coming along - this has nothing to do with staying on ESLint v8. This is just a breaking change in typescript-eslint v8.

For future reference, we [don't use gh discussions as a place to get support](https://typescript-eslint.io/contributing/discussions). Please [reach out to us instead on discord](https://typescript-eslint.io/contributing/issues#questions-and-support-requests) for these sorts of things. Thanks!
